packages/grant-explorer/src/app/wagmi.ts

```diff
@@ -1,13 +1,17 @@
 import "@rainbow-me/rainbowkit/styles.css";
 
-import { Chain, connectorsForWallets } from "@rainbow-me/rainbowkit";
+import {
+  connectorsForWallets,
+  getDefaultWallets,
+} from "@rainbow-me/rainbowkit";
 import {
   coinbaseWallet,
   injectedWallet,
   walletConnectWallet,
   metaMaskWallet,
 } from "@rainbow-me/rainbowkit/wallets";
-import { createClient, configureChains, chain } from "wagmi";
+import { mainnet, goerli, fantomTestnet, optimism } from "wagmi/chains";
```

Comment:
what are the values from this variables. I wouldn't change the RPC we used so far just to remove some lines of code

Reply:
They are the exact same. They are just exported from a different part of wagmi.
You can check that by Go to definition in your editor.
Not sure what you mean by "just to remove some lines of code"